src/main/java/com/exasol/adapter/dialects/bigquery/BigQuerySqlDialect.java

```diff
@@ -148,4 +150,13 @@ public void validateProperties() throws PropertyValidationException {
         super.validateProperties();
         validateBooleanProperty(BIGQUERY_ENABLE_IMPORT_PROPERTY);
     }
-}
+
+    @Override
+    public String getStringLiteral(final String value) {
+        if (value == null) {
+            return "NULL";
+        } else {
+            return "'" + value.replace("'", "\\'") + "'";
+        }
+    }
```

Comment:
If `\` is an escape character, then we have to replace all `\` with `\\` first.
Please add a link to the documentation for this.